crates/wdk-build/src/lib.rs

```diff
@@ -869,11 +853,18 @@ impl Config {
         }
 
         if matches!(self.driver_config, DriverConfig::Kmdf(_)) {
+            let ucx_header_path = self
+                .latest_ucx_header_path()
+                .map_err(|e| {
+                    tracing::error!("Failed to get latest UCX header: {e}");
+                    e
+                })
+                .expect("Failed to get UCX header. Please verify WDK installation");
             headers.extend([
                 "ucm/1.0/UcmCx.h",
```

Comment:
Not to be addressed in this pr, but we should track making all of these use the latest in the short term, and be metadata configurable in the medium term